### ISSUE TITLE
Can I test iFrames with Kasaya?

### ISSUE THREAD

**Issue by javaszpak** [syscolabs/kasaya#42]
Is it possible to test iFrames with Kasaya?

**Comment by hliyan** [comment]
Good question... Getting selectors to work inside iframes was part of the original plan, but I can't recall whether we already implemented that in beta. @randilu ?

**Comment by randilu** [comment]
No, we don't have access to iframes in the current implementation of Kasaya.

**Comment by hliyan** [comment]
Let's add it to the backlog for the future. Basically, our existing selectors should also look inside each iframe in the current page (so no change to syntax, if at all possible).

**Comment by javaszpak** [comment]
Thank you for the answer. BTW very nice tool!
